docs/Study-View.md

```diff
@@ -15,33 +15,32 @@ For combination chart, as long as one of the clinical attributes has been set to
+| Disease Free Survival Plot                                 	| 300                       	| This is combination of DFS_MONTH and DFS_STATUS 	|
+| Mutation Count vs. Fraction of Genome Altered Density Plot 	| 200                       	|                                                 	|
+| Mutated Genes Table                                        	| 90                        	|                                                 	|
+| CNA Genes Table                                            	| 80                        	|                                                 	|
+| Cancer Studies                                             	| 70                        	|                                                 	|
+| Number of Samples Per Patient                                 | 40                        	|                                                 	|
+| Mutation Count Bar Chart                                   	| 30                        	|                                                 	|
+| CNA Bar Chart                                              	| 20                        	|                                                 	|
+| GENDER, SEX                                                	| 9                         	|                                                 	|
+| AGE                                                        	| 8                         	|                                                 	|
 
 ## Layout
-The layout is determined mainly based on priority. Higher priority will promote chart closer to the left-top. But if the layout solely based on priority, sometimes the layout may not be displayed as expected. See the example below. 
-![image](https://user-images.githubusercontent.com/2900303/28311265-626d4cb2-6baf-11e7-8486-7873a4c42734.png)
+Study view page is fully responsive, it will try to fit more charts in the same height as many as possible base on the browser width.
 
-In order to improve the layout, we added a layout algorithm layer. The algorithm tries to fit all charts into a 2 by 2 matrix (Mutated Genes Table occupies 2 by 2 space). When a chart can not be fitted in the first matrix, the second matrixed will be generated. And the second matrix will have lower priority than the first one. If later chart can fit into the first matrix, then its priority will be promoted.
+The layout of a chart is determined mainly based on priority. Higher priority will promote chart closer to the left-top. 
 
-![image](https://user-images.githubusercontent.com/5400599/28329358-8e91312a-6bb7-11e7-97aa-a56de2abf505.png)
+In order to improve the layout, we added a layout algorithm layer. We use pie chart size as the base dimension. 1 grid will be one row * one column in the calculation. Pie chart uses one grid. Bar chart uses two grids. All charts will be placed from left to right, top to bottom. When an even size of the dimension is detected, we will promote the small charts to fit in the space. 
```

Comment:
Could you rephrase "When an even size of the dimension is detected, we will promote the small charts to fit in the space"? Not sure what "even size of the dimension" means.